src/Service/Resolvers/Sql Query Structures/BaseSqlQueryStructure.cs

```diff
@@ -122,8 +122,18 @@ public Type GetColumnSystemType(string columnName)
             }
             else
             {
+                string errorMessage;
+                if (MetadataProvider.IsDevelopmentMode())
+                {
+                    errorMessage = $"{columnName} is not a valid column of {DatabaseObject.Name}";
+                }
+                else
+                {
+                    errorMessage = $"{columnName} is not a valid field of {EntityName}";
```

Comment:
We should use the mapped field name here instead of the database column name.

E.g. id could be mapped to api_id in the mappings section